__tests__/app.test.js

```diff
@@ -90,16 +90,36 @@ describe("GET /api", () => {
                     "description": "serves an array of all users",
                     "queries": [],
                     "exampleResponse": {
-                      "articles": [
-                        {
-                          "username": "butter_bridge",
-                          "name": "jonny",
-                          "avatar_url":
-                            "https://www.healthytherapies.com/wp-content/uploads/2016/06/Lime3.jpg"
-                        }
-                      ]
+                        "articles": [
+                            {
+                                "username": "butter_bridge",
+                                "name": "jonny",
+                                "avatar_url":
+                                    "https://www.healthytherapies.com/wp-content/uploads/2016/06/Lime3.jpg"
+                            }
+                        ]
                     }
-                  })
+                })
             })
     })
-})
\ No newline at end of file
+});
+
+describe("GET /api/articles/:article_id", () => {
+    test("responds with a status code: 200 and sends a single article to the client", () => {
+        return request(app).get("/api/articles/1").expect(200)
+            .then(({ body }) => {
+                const { article } = body;
+                expect(article).toEqual({
+                    article_id: 1,
```

Comment:
Really important here that we check the article id to make sure its the correct article being returned. Well done!